### ISSUE TITLE
Cosmetic/Usability: Device editor needs add button

### ISSUE THREAD

**Issue by fidonet** [fusionpbx/fusionpbx#1112]
When editing a device and adding "Keys"/Lines and/or Settings it would be and lines it would be nice if there was an "add" button on the line to avoid having to keep hitting "save" and refresh the page.

Also the Lines dropdown runs from 1-12 and the "Key Line/Key" dropdown runs from 0 -> 99

Would be useful if they at least sync'd somewhere (or didn't start at 0 I guess)

This is specifically editing a Yealink phone, I haven't tried other device types yet.

**Comment by markjcrane** [comment]
Agreed the additional add button would be nice to have will add that as time permits.
